aea/crypto/base.py

```diff
@@ -179,27 +220,24 @@ def get_balance(self, address: Address) -> Optional[int]:
         """
 
     @abstractmethod
-    def transfer(
+    def get_transfer_transaction(
```

Comment:
Initially, I thought we can construct this entirely off-line. However, this turns out not to be possible in general as we need information like `nonce` and sometimes other ledger info beyond addresses. So we need to construct the transaction in the ledger api and introduce two performatives in the `LedgerApiMessage`: `GET_TRANSFER_TRANSACTION` and `TRANSFER_TRANSACTION`